src/lwc/geGiftEntryFormApp/geGiftEntryFormApp.js

```diff
@@ -338,16 +337,17 @@ export default class GeGiftEntryFormApp extends NavigationMixin(LightningElement
     *
     * @return {boolean}: True if card is in a 'chargeable' status
     */
-    checkPaymentTransactionStatus = (paymentStatus) => {
+    checkPaymentTransactionStatus = async (paymentStatus) => {
+
         switch (paymentStatus) {
-            case PAYMENT_TRANSACTION_STATUS_ENUM.PENDING: return false;
-            case PAYMENT_TRANSACTION_STATUS_ENUM.AUTHORIZED: return false;
-            case PAYMENT_TRANSACTION_STATUS_ENUM.CANCELED: return false;
-            case PAYMENT_TRANSACTION_STATUS_ENUM.CAPTURED: return false;
-            case PAYMENT_TRANSACTION_STATUS_ENUM.DECLINED: return true;
-            case PAYMENT_TRANSACTION_STATUS_ENUM.NONRETRYABLEERROR: return false;
-            case PAYMENT_TRANSACTION_STATUS_ENUM.RETRYABLEERROR: return true;
-            case PAYMENT_TRANSACTION_STATUS_ENUM.REFUNDISSUED: return false;
+            case this.PAYMENT_TRANSACTION_STATUS_ENUM.PENDING: return false;
```

Comment:
I think `PENDING` should be `true` here. The next step happening here is attempting the purchase call, rather than doing BDI processing. If the purchase response is successful, then the status gets updated to be `CAPTURED` or a different status if it fails.

Reply:
@bdvorachek agreed. I'll make that change.